packages/devextreme/js/__internal/ui/toolbar/internal/toolbar.menu.ts

```diff
@@ -337,6 +340,26 @@ export default class DropDownMenu extends Widget<DropDownMenuProperties> {
     });
   }
 
+  _popupKeyHandler(e: DxEvent<KeyboardEvent>): void {
+    if (
+      this._list && $(e.target).closest(this._list._activeStateUnit).length
```

Comment:
Cannot we achieve the goal without using of a list private API? It's not very good to use a private API out of the component because it'll need additional refactorings if the private api is changed.

Also I'd propose to move this stuff to a variable with a meaningful name to make this `if` more clear and declarative + simplify future debugging